alerts_consumer.go

```diff
@@ -85,7 +85,11 @@ func (c *AlertsConsumer) encodeMessage(fields map[string]interface{}) ([]byte, [
 	kvmeta := decode.ExtractKVMeta(fields)
 	env, _ := fields["container_env"].(string)
 	app, _ := fields["container_app"].(string)
-	updatelogVolumes(env, app, kvmeta.Team)
+	team, _ := fields["team"].(string)
```

Comment:
Any reason to prefer `fields["team"]` over `kvmeta.Team`? In the way you've written it, it'll use the former if they both exist, which is different from the code you had before.

Reply:
The only reason I did it that way is because `kvmeta.Team` can be set to `UNSET` whereas `team` wouldn't be. So easier to do a `team == ""` rather than `!strings.HasPrefix(kvmeta.Team, "eng-")` or something like that.

`team`: https://github.com/Clever/kayvee-go/blob/master/logger/logger.go#L322-L325
`kvmeta.Team`: https://github.com/Clever/kayvee-go/blob/master/router/router.go#L14-L21

Reply:
Good find!